File: src/main/org/nlogo/prim/etc/_behaviorspacerunnumber.java
package org.nlogo.prim.etc ;

import org.nlogo.api.LogoException;
import org.nlogo.nvm.EngineException;
import org.nlogo.nvm.Context;
import org.nlogo.nvm.Syntax;

public final strictfp class _behaviorspacerunnumber
	extends org.nlogo.nvm.Reporter
{
	@Override
	public Syntax syntax()
	{
		return Syntax.reporterSyntax( Syntax.TYPE_NUMBER ) ;
	}
	@Override public Object report( Context context )
	{
		return report_1( context ) ;
	}
	public double report_1( Context context )
	{
		return workspace.behaviorSpaceRunNumber() ;
	}
}

File: src/main/prim/etc/_behaviorspacerunnumber.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.core.{ Syntax }
import org.nlogo.nvm.{ Context, Reporter }

class _behaviorspacerunnumber extends Reporter {
  override def syntax =
    Syntax.reporterSyntax(
      ret = Syntax.NumberType)
  override def report(context: Context): java.lang.Double =
    Double.box(report_1(context))
  def report_1(context: Context): Double =
    workspace.behaviorSpaceRunNumber
}
